tasks/build-examples.js

```diff
@@ -82,7 +82,9 @@ function augmentExamples(files, metalsmith, done) {
       if (!(jsFilename in files)) {
         throw new Error('No .js file found for ' + filename);
       }
-      var jsSource = files[jsFilename].contents.toString();
+      var jsSource = files[jsFilename].contents.toString()
+          // Change data paths to absolute urls
+          .replace(/'data\//g, '\'http://openlayers.org/en/v' + pkg.version + '/examples/data/');
```

Comment:
That's done already @tsauerwein. See https://github.com/openlayers/ol3/blob/v3.16.0/tasks/build-examples.js#L115-L117.